README.md

```diff
@@ -0,0 +1,179 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">Ruby</a></li>
+    <li><a href="https://rspec.info//">Rspec</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- _1 - Users can register and log in, so that the data is private to them._
+- _2 - Users can introduce new transactions associated with a category._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://rails-kbsp.onrender.com/)
+
+- [Loom Video](https://www.loom.com/share/e64e4e5968e840c9a7ce23b0c46ab22d)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install bundler
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/adarachel/budgetapp.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd budgetapp
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤  **Ada Rachel**
+
+- GitHub: [@adarachel](https://github.com/adarachel)
+- Twitter: [@adarachel_dev](https://twitter.com/adarachel_dev)
+- LinkedIn: [@AdaRachel](https://www.linkedin.com/in/adarachel/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- _Implement the left side menu to improve the navigability of the app._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐ Show your support <a name="support"></a>
+
+If you like this project, please give it a star ⭐.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Thank you to Microverse.
+
+>  Thank you to [Gregoire Vella] (https://www.behance.net/gregoirevella) on Behance, the author of the [original design] (https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
+
```

Comment:
[OPTIONAL] The `Gregoire Villa` and `original design` texts are not showing as a link. I think it is because of the space between the square and round brackets.